include/hpp/core/collision-pair.hh

```diff
@@ -56,10 +56,10 @@ struct CollisionPair {
     //decltype(computeCollision(tf1,tf2,request,result))
   {
     using ::pinocchio::toFclTransform3f;
-    assert(!first ->getTransform(d).translation().hasNaN());
-    assert(!first ->getTransform(d).rotation   ().hasNaN());
-    assert(!second->getTransform(d).translation().hasNaN());
-    assert(!second->getTransform(d).rotation   ().hasNaN());
```

Comment:
Shouldn't you keep the `d` here ?

Reply:
You are probably right. I pushed a correction.